src/main/resources/db/migration/V0__initial_version.sql

```diff
@@ -0,0 +1,41 @@
+CREATE SCHEMA tasker;
+
+CREATE TABLE tasker.boards
+(
+    id          uuid PRIMARY KEY,
+    title       varchar(50),
+    created_at  timestamptz,
+    description varchar(2000)
+);
+
+CREATE TABLE tasker.columns
+(
+    id       uuid PRIMARY KEY,
+    title    varchar(50),
+    board_id uuid REFERENCES boards (id) ON DELETE CASCADE
+);
+
+CREATE TABLE tasker.cards
+(
+    id          uuid PRIMARY KEY,
+    column_id   uuid REFERENCES columns (id) ON DELETE CASCADE,
+    board_id    uuid REFERENCES boards (id) ON DELETE CASCADE,
+    title       varchar(50),
+    description varchar(2000),
+    priority    int,
+    due_date    date
```

Comment:
Why not `timestamp with time zone`?

Reply:
I don't think that due to should be that specific with timestamp. Just date would be fine in my opinion. For instance, card's due date is 29 July 2020.
what do u think ?

Reply:
what about time specific ?

Reply:
ok, that is fine I can change it to timestamp too